src/views/DiscoverProjects/Actions/JoinProjectModal/index.tsx

```diff
@@ -0,0 +1,135 @@
+} from '@togglecorp/faram';
+
+import DangerButton from '#rsca/Button/DangerButton';
+import Button from '#rsca/Button';
+import TextArea from '#rsci/TextArea';
+import Modal from '#rscv/Modal';
+import ModalBody from '#rscv/Modal/Body';
+import ModalFooter from '#rscv/Modal/Footer';
+import ModalHeader from '#rscv/Modal/Header';
+
+import _ts from '#ts';
+
+import {
+    ProjectElement,
+} from '#typings';
+
+import styles from './styles.scss';
+
+interface JoinProjectFaramValues {
+    reason: string;
+}
+interface JoinProjectFaramErrors {}
+
+interface JoinProjectParams {
+    projectId: number;
+    projectName: string;
+    body: string;
+}
+
+interface Props {
+    project: ProjectElement;
+    onProjectJoin: (params: JoinProjectParams) => void;
+    closeModal: () => void;
+    projectJoinRequestPending: boolean;
+}
+
+function JoinProjectModal(props: Props) {
+    const {
+        project: {
+            id: projectId,
+            title: projectName,
+        },
+        closeModal,
+        onProjectJoin,
+        projectJoinRequestPending,
+    } = props;
+
+    const [
+        joinProjectFaramValues,
+        setJoinProjectFaramValues,
+    ] = useState<JoinProjectFaramValues>({});
+    const [
+        joinProjectFaramErrors,
+        setJoinProjectFaramErrors,
+    ] = useState<JoinProjectFaramErrors>({});
+
+    const [pristine, setPristine] = useState<boolean>(true);
+
+    const handleFormChange = useCallback((faramValues, faramErrors) => {
+        setJoinProjectFaramValues(faramValues);
+        setJoinProjectFaramErrors(faramErrors);
+        setPristine(false);
+    }, [
+        setJoinProjectFaramValues,
+        setJoinProjectFaramErrors,
+        setPristine,
+    ]);
+
+    const handleValidationSuccess = (faramValues: JoinProjectFaramValues) => {
+        onProjectJoin({
+            projectId,
+            projectName,
+            body: faramValues,
+        });
+    };
+
+    const handleValidationFailure = (faramErrors: setJoinProjectFaramErrors) => {
+        setJoinProjectFaramErrors(faramErrors);
+    };
+
+    const schema: Schema = {
+        fields: {
+            reason: [requiredCondition],
+        },
+    };
+
+    return (
+        <Modal
+            className={styles.modal}
+            closeOnEscape
+        >
+            <ModalHeader
+                title={_ts('discoverProjects.joinProject', 'title', { projectName })}
+            />
+            <Faram
+                schema={schema}
+                onChange={handleFormChange}
+                value={joinProjectFaramValues}
+                error={joinProjectFaramErrors}
+                onValidationSuccess={handleValidationSuccess}
+                onValidationFailure={handleValidationFailure}
+                disabled={projectJoinRequestPending}
+            >
+                <ModalBody className={styles.modalBody}>
+                    <div>
+                        {_ts('discoverProjects.joinProject', 'requestQuestion', { projectName })}
```

Comment:
```suggestion
                        {_ts('discoverProjects.joinProject', 'requestQuestion', { 
                            projectName: (
                                <span className={styles.projectName}>
                                    {projectName}
                                </span>
                            ),
                         })}
```
lets do this and make the project title bold